db/db.go

```diff
@@ -124,7 +170,7 @@ func (db *Database) stateS3toDB(sf *statefile.File, path string, versionID strin
 
 		st.Modules = append(st.Modules, mod)
 	}
-	return
+	return st, nil
```

Comment:
```suggestion
	return
```

should suffice since `err` is `nil at this point